lib/query_components/base_component.rb

```diff
@@ -2,14 +2,13 @@ module QueryComponents
   class BaseComponent
     include Elasticsearch::Escaping
 
-    attr_reader :search_params
+    attr_reader :search_params, :search_term
 
     def initialize(search_params = SearchParameters.new)
       @search_params = search_params
+      @search_term   = @search_params.query.strip.sub(/^"/, "").sub(/"$/, "") unless @search_params.query.nil?
```

Comment:
@rboulton instance variables are nil by default, so this code does the correct thing.

@srmoscow are we sure we need to strip the quotes? It may be that elasticsearch strips the quotes from the query anyway, so perhaps this change is not necessary.